src/components/com_tjcertificate/administrator/views/certificate/tmpl/edit.php

```diff
@@ -63,3 +87,41 @@
 	</form>
 </div>
 </div>
+
+<!-- Modal -->
```

Comment:
I think this code can be reused, can you create a preview layout and use it in both template & certificate

Reply:
done